bundles/org.openhab.binding.meross/README.md

```diff
@@ -0,0 +1,95 @@
+# Meross Binding
+
+This binding integrates **Meross**&reg; devices
+
+## Supported Things
+
+Supported thing types
+
+- bridge
+- light
+
+|   Meross Name       | Type   | Description         | Supported | Tested| 
+|---------------------|--------|---------------------|-----------|--------|
+| Smart ambient light | msl430 | Smart ambient light | yes       | yes    |
+| Smart plug          | mss210 | Smart plug          | yes       | yes    |
+
+Anyway, the togglex capability may consent to control on/off of the majority of plugs and bulbs
+
+## Discovery
+
+The Discovery service is supported.
+if a refresh of devices is needed, e.g. to fetch new devices please disable  and re-enable the bridge via the interface button 
+
+## Binding Configuration
+
+To utilize the binding you should first create an account via the **Meross**&reg; 
+**Android**&reg; app. 
```

Comment:
there also seems to be a iOs app, is that also supported?
```suggestion
To utilize the binding you should first create an account via the Meross Android or iOs app. 
```

Reply:
I had a look at the Meross, and yes  IOs in also supported so I would mention it too